### DISCUSSION TITLE
firefox: using firefox-common.profile blocks sound

### DISCUSSION THREAD

**Discussion by xeno-crimson** [netblue30/firejail#5580] [Q&A] [answered]
~/.config/firejail/firefox-common.profile
```
# This is my firefox-common.profile

# Add the next line to your firefox.local to enable native notifications.
dbus-user.talk org.freedesktop.Notifications
# Add the next line to your firefox.local to allow inhibiting screensavers.
dbus-user.talk org.freedesktop.ScreenSaver
```

I fixed the issue by renaming it to firefox.profile but I am curious about why its happening

**Comment by rusty-snake** [comment, marked as answer]
This will greatly decrease your security. Don't do so! Name it <code>firefox-common.<strong>local</strong></code>.

**Comment by rusty-snake** [comment]
And to answer your question we need more details.

What program are you even trying to use? With which firejail version and which distribution and with which sound system? When did this started? Does it happen with `--noprofile`? ...

**Reply by xeno-crimson** [reply]
I was trying to start firefox but kept giving no sound but your above comment fixed the issue for me.

**Comment by ghost** [comment]
Both comments refer to `firefox.local` but you put them in `firefox-common.profile`. If you name something in ~/config/firejail with .profile without including the corresponding one from /etc/firejail, the latter will not be part of the sandbox options. See https://github.com/netblue30/firejail/wiki/Creating-overrides for details on how to do overrides correctly.

> I fixed the issue by renaming it to firefox.profile but I am curious about why its happening

Same problem here. Audio might be fixed but other options from /etc/firejail/firefox.profile are likely missing, resulting in a weaker, less secure sandbox. You can follow the included profile chain by running `firejail --ignore=quiet /usr/bin/firefox` from a terminal.
